website/docs/r/servicebus_queue.html.markdown

```diff
@@ -42,56 +42,45 @@ resource "azurerm_servicebus_queue" "example" {
 
 The following arguments are supported:
```

Comment:
Arguments are sorted according to <https://docs.microsoft.com/en-us/azure/templates/microsoft.servicebus/2017-04-01/namespaces/queues>